pymmcore/__init__.pyi

```diff
@@ -1014,10 +1014,22 @@ class CMMCore:
         """Sets the current slm device."""
     def setSLMExposure(self, slmLabel: str, exposure_ms: float) -> None:
         """For SLM devices with build-in light source (such as projectors),
-
         this will set the exposure time, but not (yet) start the illumination"""
+    @overload
+    def setSLMImage(self, slmLabel: str, pixels: np.ndarray[np.uint8]) -> None:
+        """
+        Write a 8-bit grayscale image to the SLM. Pixels must be a 2D numpy array [h,w] of uint8s.
+        Warning: SLM might convert grayscale to binary internally.
```

Comment:
```suggestion

        !!! warning

            SLM might convert grayscale to binary internally.
```

This will render more uniformly like the other warnings in the pymmcore-plus docs which uses these stubs ([example](https://pymmcore-plus.github.io/pymmcore-plus/api/cmmcoreplus/#pymmcore.CMMCore.addSearchPath))